nextmv/nextmv/cloud/application.py

```diff
@@ -4,6 +4,7 @@
 import random
 import shutil
 import time
+from collections.abc import Callable
```

Comment:
Hmm, I always use this:

```python
from typing import Callable
```

Which one is preferable? :thinking:

Reply:
```suggestion
from typing import Callable
```

:point_up:  Just in case you want to quickly replace it. But again, ignore if this is the more fitting type. :relaxed: